services/sentry.py

```diff
@@ -70,7 +70,7 @@ def parse(data):
 
     # tokens with values we want to strip
     tokens = [
-        "GITHUB_TOKEN",
```

Comment:
It might be worth keeping this value in just in case. I don't often say that, but in this instance I think we want to be very sure this is always stripped even if we shouldn't be using a token with that name.